internal/agentsession/capture.go

```
@@ -81,6 +81,381 @@ func Capture(sessionStore, cwd string, launchedAt time.Time, claimed map[string]
+	return !seen || activity > prev
+}
+
+func snapshotCodex(root, cwd string) (map[string]int64, bool) {
+	if root == "" {
+		return nil, false
+	}
+	wantCwd := resolvePath(cwd)
+	snapshot := map[string]int64{}
+	_ = filepath.WalkDir(root, func(path string, d os.DirEntry, err error) error {
+		if err != nil || d.IsDir() {
+			return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 153) but it returns nil

(nilerr)